json_schema/module/biomaterial/collection_institute.json

```
@@ -0,0 +1,44 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "Information about collection institute location.",
+    "additionalProperties": false,
+    "title": "Collection institute",
+    "name": "collection_institute",
+    "type": "object",
+    "properties": {
+        "describedBy": {
+          "description": "The URL reference to the schema.",
+          "type": "string",
+          "pattern" : "^(http|https)://schema.(.*?)humancellatlas.org/module/biomaterial/(([0-9]{1,}.[0-9]{1,}.[0-9]{1,})|([a-zA-Z]*?))/familial_relationship"
+        },
+        "schema_version": {
+          "description": "The version number of the schema in major.minor.patch format.",
+          "type": "string",
+          "pattern": "^[0-9]{1,}.[0-9]{1,}.[0-9]{1,}$",
+          "example": "4.6.1"
+        },
+        "latitude": {
+          "description": "The latitute of the collection institute, in decimal degrees.",
+          "type": "string",
+          "pattern": "^-?([0-8]?\\d|90)(\\.\\d{1,6})?$",
+          "example": "52.1; -0.127541; 38.045956",
+          "user_friendly": "Collection institute latitude",
+          "guidelines": "Use one decimal place to keep the exact location private."
```

```suggestion
          "guidelines": "Only specify one decimal place if the exact location is considered sensitive information"
```

Just a suggestion, the current wording might cause submitters to only specify one decimal place even if the location isn't sensitive.

---

Thanks for the suggestion, I changed the wording to 
> If the exact location is considered sensitive information, use one decimal place.

in commit 53689b766950d53528dc36b7dbc25fb0b835c59e .